package.json

```diff
@@ -5,19 +5,20 @@
     "type": "git",
     "url": "https://github.com/mobify/split-test"
   },
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
My general feeling is that MAJOR MINOR PATCH more applies to describing changes to the API, rather then the build tools. For this change, yes, it's technically backwards incompatible, but not at the API level, so I'm okay with not bumping the major. Feel free to disagree :)